examples/t8code_2d_dgsem/elixir_advection_unstructured_flag.jl

```diff
@@ -61,10 +61,15 @@ save_solution = SaveSolutionCallback(interval = 100,
 # time step.
 stepsize_callback = StepsizeCallback(cfl = 1.4)
 
+# The SaveRestartCallback allows to save a file from which a Trixi.jl simulation can be restarted,
+# possibly with a different polynomial degree.
+save_restart = SaveRestartCallback(interval = 100,
+                                   save_final_restart = true)
+
 # Create a CallbackSet to collect all callbacks such that they can be passed to
 # the ODE solver.
 callbacks = CallbackSet(summary_callback, analysis_callback, save_solution,
-                        stepsize_callback)
+                        stepsize_callback, save_restart)
```

Comment:
```suggestion
                        save_restart, stepsize_callback)
```
That's the usual order we introduced for consistency. First, we save all the stuff belonging to the current time step. The step size callback belongs more closely to the next time step, so it should come last. See https://trixi-framework.github.io/TrixiDocumentation/stable/callbacks/#Usage-of-step-callbacks